<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotoApp</class>
 <widget class="QMainWindow" name="VotoApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>429</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VotoApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Notas al margen</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditFree">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButtonGen">
       <property name="text">
        <string>Generar voto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditDST"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ubicaci√≥n de destino</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButtonDST">
       <property name="text">
        <string>Guardar en...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>(opcional)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAVgAAACSCAMAAAA3tiIUAAAA6lBMVEX///8AkTxKqisAAAD7+/s1
owC73ccAiSjr9OkAjzdJqydApWTz8/P5+fnk5OTh8NzZ2dna7+Pi4uLp6env7+8AjTBoaGjDw8PN
zc0VlEL5/vxtbW2AgICoqKhlZWXa2tqfn59XV1c/pxm+vr5PrDJ0dHSLi4uYmJi2trakpKRPT09I
SEhcXFyCgoLJyckdHR0PDw81NTUsLCw7OzsuLi7U6c4ZGRkAhRduuVh5vmVcskPt9+osnlZNqGu3
266k0LOdzo+EwJfJ5dSDwnButoXa7tWg0JOMxn1cr3gAggCs1aCWyKV7vI/E4byfCDdJAAAQ1UlE
QVR4nO2dC3ubRhaGJxaJ7BQIAQzEXAQmXAQSujlOm8u23W7b3Xb9///OzjkzIBCyLSWy5U3me57G
MMzA8M6ZMzc0JURISEhISEhISEhISEhISEhISEhISEhISEhISEhISEhISEhISEhISEhISEhISEhI
SEjoCev01TZdEHLxersujp3j/wud/nJyuUXPPxLy4v35Vp39enrsXD99vbu8PNkmBDt8tl3nw38e
O99PXa8uX27levL89A6wz569f33snD9xfbiF631gz94IR3uXbp4zjC97fO8B+2z44th5f9L6Ex3s
5dvfftpEex/Y838dO+9PUb/f3Nz8DgdvkesftFqf/vhyL7BnP0D61y9evBDettGH59Cf+o0eAczL
PzHw4u3+YP/9fjgcvv8k3C3TBXOsz19xsO9Y8B+X+4J9/Z55BdGrpVCpTi+7YF+xSzf7gx1ysHDX
xmwl+I+qOZVQpAlqLpF1iFT/ldq3kEh9Wl9Yp+3fvUn6+Hr34S3Vh5Mu2Bt28ZcvBfvsDegTjhmU
0LVCEk+sICS5S9/STMjEtaxgLAVWkMqEyKEVZDI+0Q8s11fwbxBDQBZYE43oXpDINMyyEuKNaHBi
0sSWpxESBi6Nn+Y0MIcjyVJSC0SmNKlORl7gmUcAe9JuoGqwL3/Cax/36xW0wbKA/9CbSBNfkSop
D5HbxKdgXXoUUmxyIJEwozHGkuS7+MgsBswKCTWevylgVIoR0XVCNI+eJHDJomAVok0oT9cgRE2A
aU6Ljt6NFpEzhYg0BZELjYzUxyTK1G2farAnl7+9Ihfv3u7fK+iAfTYEghk+iIONXb0NlhBjSsYY
A22OZDbEylpggY45YSddsFDFIxXB+nbm0Gf4yWgTrDp5HJCbugXsycvLD283h7ZfBjZg9TCfOpVK
wY6SNljVSGMyRYh2Cv8i2JFLxr5jKxCQ5lWlkOkUja4PNq0QrKVUKYBVLQ4WI+aVLZGpdwR7vR3s
yUl/YPtFYKWAOc/cDalR+TlJY7kBO/GKkBAPwVZrsLpF/GmYYYmkKR7kkzGpwYJzqMHS+BSsk9Ln
mBQsCX3SgPVoUhmS+o+OlfDxwDawfe0P9j0gYJWau4KcyJbRcgVGJJPUXkdAsNV0wxVQKV6+zWJd
DcCGE9eNbACrBEbQWCw3VSWJHwHkpn58Dro8NNgh6P0b+oAqwufkYHEAlthuQtaNV5wRrZShfUID
BbBmoW+ABaMH8AjWpwdGRBHS4zG9lWsSS5aIFgBY4iRW42NVntQ/hslevENdHhbs2Yt/grAj60ee
Z0n5ZOrRNglaqATak4weyRE1ukgjMY1ROJidzJomE+hEuZ4XQsA08ZKY+owkMIAaoApcjyZSCs8L
qPvwC495EWsEyEkKxYQRE5rUNiDpMbpbTBcHA8tHXu0hrawaCv1HVQ0igwEpFBEx4QgP6Fsrqqqw
uCbEwjT8gAaoMo2AJ5iSJsPYBlyACAbei5gKu7taRzTaSY+ki+cvUadfC/biDDUUcwVc//gAggHX
14ElLz6Bfj72+zxBfSVYodv07YE1nGPnAPXNga0Gg1Q5dibINwdWTgdUxxghbOrbAutfU6yr6tjZ
AH1LYI0rMNejTBX09S2BNQeD+VPwAqgnB1ahIyhDvjdab4kGlOWqsUPabZLhsRuNnmTI5hcPjw8O
Vp+tyiKaBFYwKcrVYkVzpi2L6KrTCRp1I5XNGznFHKrzYIWW5y8m09CPHV3XbD9LillFxsvF4mq5
nFEto7xDQreu5uu0KMUfJ1ejdqTQz0MaYNIsLcKGYL7Ep86DTtwMwvac3v14enr68RBgT0HtO2uD
rmhL4sBf+55IILNYB81oSSQb0Whzn3YDsjXDoJW2br4MPGvhN7kXVuFvygPt63XSoBV5AQHTvbj+
iR9q/vL1YH8dwlRB+5sYfYNFzsF2mmpjIxJrcEabKaM+WH8jJKjvON8GXMaT1nKNzLNktMBmnZTz
pvJXeL7ch+vGdwVfDpbPbg1bNou2MHMT15pEtJovc26fHbAKZnkdCcGa/NWWC37gzOg/10uspovl
NSKJeZTFkkeadkpquZi3ySrsbNwDa67B8qKaL67YQWPgQbvQdwR7eSiw/LuC1uwWvmJ3VDnqBUn9
SIQARu4gHfQJfPYbWdZNFZjRnJ2ozFPA1LiCaRe4QKkxIHjMwgeDxnUq/JLSwGdeyUVXit3gBX9U
Xc7FTkhf/fEj6OVhwT779w9Uv75u8qN3HqpvB7sxcRpCmFWfZS0XCBdqM9LatTOszXEKB0kdnDeG
J3ErXDaeE87iFlhp0bZKcPL1YgZ4iLJTKnfp7SVOxJ4cGCzOyp7DKi2rbF2wasv8WmC7PRkMKluc
Z3W/qWqDhUJa1HEUXqHxAcE6bd5U9BU3O7cN1uZ/Q8JLrdVAlXVxYlJ9MVgX8N1gO+geYDGRAeqC
NXtgSR+svWnWTT/VaYM12mDRDXqEuBCj3X8FJ3GNnIB4bdf1gwHsnIcVg1vaJxsvQBHN7oNK7lr+
PgTYYZP1Lli5X6H6YKEHsNqaaaftY+VONA/BYll6vSeClwVuMlZodf1gaEevmAOQG9velIu+wmzc
9d16LLCOpFDJsmyCI91ixGAwWjuSNNuE00hrgyUdsBZiGfWLKeDVH4pLQXrXUvNgqBclA6v3s8Y0
4pUAim6Xr2se3hUg2Pns+vp6Pp9zS2lZDBd2ydeRlLWN3QK2OWuDlVnDs6W+htxhTxCPtvbBSw62
YI+7tarDQARW8rG67LA++eEBG6/heQO2JZkHdTM368SZcz+8fe7/VrCSy+wtG3SaPcJ5LBqwrPuA
TT+0RuDuIwY2vMUBSVAA2PO7GuzUlf3410+gg3e3Pn/+/AP/UV2X63Ud1LXYLtjVfmBn49x2HDud
cwcJYDd6mwD2ag0W/yLQVRdsdgtYu7YJHD/sPvq6PCzYzncFSCpwqawgWkW3g20iLSwOdvsMdQ9s
W/j9Up9OzINqsDI8EAYWJQfLXcF40O5krAWulfWLccCzU1eWNN8VHGJIy5xAexqmT3GLK5gPuv0j
5i4zsk2dxkvqcsUG3e6QR8GIAcq0BstuUjCgAJY3XmvL7AiLmU+SJYNWN/g+/YHfFfxFvnoS5m/8
ruDX9r2Z3yObQUYvpNuOg1Pb3vx2ulv4yvPl1RWbK8ASVPtuBLhBt79ouGWs5GqLXQ3Ws1ydiTcU
zh+Ume/742S+nf19Ovh8rNIHq+wCFnpOs63Lq72RF5uAwuBpc7egnUSrUa/Wz8Hph1HJL4BrGPOK
kpBNtaYvmXboym7o4GC3zBXIPbBbhrTOZv6boVrcBgvIZtL67dGU0k0/WHLXiXMFRuuZsxkHCych
4VNYLc9lQOyN+cvBnpOHqIODNfpgze1gN3qH190wrxkE5+26iLNbjLLNKjfhJXclt9Kyis5mt+rM
ODUlh2cgqwPXfTVjCeUR9sBuHUTcqYOD7c+4MNad9kzuR2L8lvwFDNrqzMzWhZpaq7XByVbmPaBx
H6y4zSpJA0uet025ntF22tOGaPe1c69m6BjAgZS2bVdVRf/Nt7uLe3T4NS/IhqsZhjpyYj9MU4m3
EDRE1xw797O0FwnHsmy9wIt1w57OmxfnLUntN/LWSVjXZ95PHSS5blQpa9aw2DZmKYIG7HqOgE2R
LzLH0HOLGWe7TwBCT3N0sJvLWbP+as2gF2mOSRfdwIL71XEb7Lhl/ciET7VuNjesOmxUH2lWg22t
IDgbKcfoSWatRWDM7b7N14ODtfpgl72XYbMvStkOa5p5ZFk7X7TS2gZxfpuPN90tXNmTWy5Hr8G2
17yqTsqc9Rg6VR+/rzk22E1mWZ91wTxlS/XE3brVmK8tpLMKnbZRYet9xU/y9VrrtDa3pttVCz1J
xU25nkyT14VSqNyNdwwUnfaezdfh+7FxbFeOo41GmlPFoUXfQsljG8Lo+D6n/tStNiJl1rqLHlvl
YlEmbRhqTONIrRO/af8dm15qOr9aUiwWq6DFRIkrJ+582VHRAPilKFxYO18li1aLVZGiw9HzPO9+
UIOxdwN7cXp6enEIsM2NhEB/ncDc4SE+lX8Gm0X9fez3eSq6eI6/Tb78+NU/7hiewZH4cQfXQ35X
8J3q9B+oA090n/39M+h73mcDN4Opt4Y73ArC+fnwfDh8c+y3O6IeYTHxu9RjLH9/lxJgH0qP4Qqk
cZalprbzOhGBj9F22HvIjNjPvO+S3l5IyL7qBwmj4P44az3kR3Fn3GKlIq9s5Tawo20rhma5JRC2
g+hEKoi8sVwabszpEr29bLYvWKPzrdF+YH//6UG+K/iM4lvKShGM5WuwKrCAPYkknYwMIo+TkaRL
I4nIWj2xoo7MgkdaJ5E1mTjlCL8/0unoXtWpxcoqXKAjekMxKVPVyunVEX4hq+EEAlqszvFntkmf
Qy/hg/D2cGuFPl7hT1MxmYEJlNylYQrejl7mt8Kf9Gu7buZ1eViw3QGCFOmyWYP1IndM0kngEbmY
Jgtdm0Su7E4DsyoT2CGGyi8TLyJkGgTMQLPITYhWJJHsFa6WUwsPVBIWbjIhVUDUIik1kqZJ5NGY
gU88a5IRtUwihYGVJ5Y1QcpZ6kYJkaOkzIliBYFN/MhKqClOAze2Sh8eBTfzIY+wA1KUKHqZTBKW
B5dIbuAWKskjd/tXZT095KfysBXWZBJxsD4so8KWLelILk2SJ7ADDLFyWvmprRVQk0cUqlPitka4
/wskTO0MXlUu2B4mrqFSX+FQH+sS2Iono8wIKRVIAOAL2ee1mIJN/XqTKohUyJT3yCLjkLDNo1Jb
XUjETcFrOBa9VpKVyr4Z1Wk+Yb8PN3YC2NaLwF3jhEx2nzH8DbeGgb0iv3IS5r+4aWR3w9O2K5jC
/F9YluXCkSe4E5RNc2+pbH8cdIJAVC7JtKCR4FOYMT1Y+aZrOeh6Gdg4ZY2XS4xp4mawTxSxDJI4
JKM3v9Jk3AEFwUa04qv4AQ7c3jPlNPFctmNMTOOucp3mzKvgdmOYYVxIVZSyjzoSooCzjzPY0obe
yhqhZ9eKdOdtO17d3Nzgjoa3bnh8P9jPkH7LNqdSZDZgU0CHm7cQBJuQmFqsqxINWhkPsFcUsV6S
rP7yLAfjk8CUdLRYcAUGRFIRLAVoZ7gVEoDVcLcZRFqqDCwtEFIlDVjZjYkRsE2pWKdiBGBtAOvT
eyuwNJBjQwWb+8AWM6EPD6VW68Xca+fRrmAbdfcz3AfsrRvz0l6BHZsa/pxAK/I4NsvcThU5wnfS
ylgJaN1K0mqMP6oyC99OSuolczuDGimXeZUqdmwXqrLIVa20/ZWhrGikCOpuMLZdBjYwqKvVtCim
Nl/lVQFWpUfEjuI40mqwiUwf5FnEmcS5phR+lZrQ3Cc2dPHkKK+8kIQV2xFQL2ISJnZeypAZN6Bl
G9tWTPwq338DunfPvxTs+9u2kpb8LMtUg1mSFoa0NwC7mynUJCEwz5QcdtzJM74SoIfhaAwxM7Zu
oMLBKIOuVJWp1D5tGp9G0nyi58QIfccmMPtPkxuZTbQsHEEaZGnCllEh74XZ9E8uy+NQh9AsNGhi
emuDnuUjIvsSMcYAP87GbBUiH8MGVmMDMjOmj+N5qjL/C375+cWbn3/a/1nflU77257vArb7haHQ
Fr36sP3/hHAX2LPhG7Ex9726+OvHk7d9XQLY8zdb9elvsWawky626dYLF4KqkJCQkJCQkJCQkJCQ
kJCQkJCQkJCQkJCQkJCQkJCQkJCQkJCQkJCQkNDx9D/Z8dbYDkWaYAAAAABJRU5ErkJggg==&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rellena esta papeleta y deposita tu voto en nuestra urna virtual.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Clave p√∫blica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPEM"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPEM">
       <property name="text">
        <string>Elegir...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;¬øQuieres que la Lista A: Ampliando Electr√≥nica&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;sea elegida como CEE para el per√≠odo 2020?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxApprove">
         <property name="text">
          <string>Apruebo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxReject">
         <property name="text">
          <string>Rechazo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
